privly.safariextension/scripts/popover.js

```diff
@@ -43,6 +43,26 @@ function deactivateExtension() {
   extensionStateChange("#activateExtension", "#deactivateExtension");
 }
 
+/**
+ * Helper function to open new tabs from the links.
+ */
+function openPopoverPage() {
+
+  /* istanbul ignore if */
+  if (typeof safari !== "undefined" && safari.application !== undefined) {
+    safari.application.activeBrowserWindow.openTab().url = safari.extension.baseURI + this.getAttribute("data-popover-path");
+  }
+}
+
 // Set the activation UI
 $("#deactivateExtension").click(deactivateExtension);
 $("#activateExtension").click(activateExtension);
+
+// Open new windows from the links
+var links = document.getElementsByTagName("a");
```

Comment:
Add a CSS class to the relevant elements and use `$(".the_css_class_here").click(openPopoverPage);`